frontend/src/component/common/ConstraintsList/StrategyEvaluationItem/StrategyEvaluationItem.tsx

```diff
@@ -49,10 +57,11 @@ const StyledType = styled('span')(({ theme }) => ({
 export const StrategyEvaluationItem: FC<StrategyEvaluationItemProps> = ({
     type,
     children,
+    alignTypeCenter = true,
 }) => {
     return (
         <StyledContainer>
-            <StyledType>{type}</StyledType>
+            <StyledType alignCenter={alignTypeCenter}>{type}</StyledType>
```

Comment:
Not sure I understand why this prop is useful. I saw you used it in the variants split preview, but I don't understand why. I see what it does, but I don't see anything in the screenshots that suggests it's necessary? The variants text is also left-justified in the screenie, isn't it? Is it ..missing padding or something?

On a secondary note, if this is useful, wouldn't it be better to make it an enum, so we don't end up with a bunch of booleans? E.g.

```tsx
typeAlignment: 'center' | 'start' = 'start' // or 'center' | 'left'?
```

or something?